e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -658,14 +691,16 @@ async function performImageryViewOperationsAndAssert(page) {
   const mainImage = page.locator('.c-imagery__main-image__image');
   await expect(thumbnailImages.first()).toHaveAttribute('src', thumbnailUrlParamsRegexp);
   await expect(mainImage).not.toHaveAttribute('src', thumbnailUrlParamsRegexp);
-
   // Click previous image button
-  const previousImageButton = page.locator('.c-nav--prev');
-  await previousImageButton.click();
+  const previousImageButton = page.getByLabel('Previous image');
+  if (!(await previousImageButton.isVisible())) {
```

Comment:
The previous image button doesn't appear until you hover over the object view. If the button is not visible, then we tell playwright to execute a hover action to make it appear. Otherwise, it just clicks it. I did try removing these lines and the tests fail without them

Reply:
ok, do you mind putting a comment above the if statements